### PR TITLE
Implement MinigameService

### PR DESCRIPTION
**Pull request by kinggoesgaming**
Implements MinigameCore/MinigameCoreAPI#18

### CODE REVIEW

File: src/main/java/io/github/minigamecore/plugin/MinigameCore.java [hunk 1]
```diff
@@ -57,15 +61,30 @@ private MinigameCore(@ConfigDir(sharedRoot = false) Path configDir, Injector def
...
+    /**
+     * @deprecated Only used for initial injector setup.
+     *
+     * <p>
+     *     Use {@link MinigameService#getInjector()} instead
+     * </p>
+     *
+     * @see MinigameService#getInjector()
+     */
+    @Deprecated
```
Comment (Flibio) [inline src/main/java/io/github/minigamecore/plugin/MinigameCore.java]: Just remove this method, and pass the default injector into MinigameService via the constructor.

Comment (kinggoesgaming) [inline src/main/java/io/github/minigamecore/plugin/MinigameCore.java]: thanks for noticing... i changed the way i was handling that but forgot to remove the method :stuck_out_tongue_winking_eye:

**Comment by Flibio** [comment]
:+1:

**Comment by Flibio** [comment]
Oops, want to reopen this PR?

**Comment by kinggoesgaming** [comment]
#28

**Pull request merged**